pull-request/pull-requests.bash

```diff
@@ -5,6 +5,9 @@ set -e -x
 # SANITY!
 export TZ=UTC
 
+# Be able to refer back to the script
+selfpath=$(readlink -f "${0}")
+selfdir=$(dirname "${selfpath}")
```

Comment:
There's already `SCRIPT_DIR` for this purpose.

Reply:
done